grpc/python/BUILD

```diff
@@ -38,13 +39,13 @@ python_grpc_compile(
 python_repackage(
     name = "protocol_pkg",
     src = ":protocol_src",
-    package = "grakn_protocol"
+    package = "graknprotocol"
```

Comment:
Nexus Repo dislikes underscores in package names. Python dislikes hyphens in package names.